src/unix/utilsx11.cpp

```diff
@@ -2680,7 +2680,7 @@ bool
 wxDoLaunchDefaultBrowser(const wxLaunchBrowserParams& params)
 {
 #ifdef __WXGTK__
-#if GTK_CHECK_VERSION(3,90,0)
+#ifdef __WXGTK3__
```

Comment:
We can't remove the version check completely, this function is only available in 3.22.

We also presumably will need a run-time check if we use it with GTK+ 3 too.